plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -49,9 +51,18 @@ public static QueryRunner createSqlServerQueryRunner(
             TestingSqlServer testingSqlServer,
             Map<String, String> extraProperties,
             Map<String, String> connectorProperties,
-            Iterable<TpchTable<?>> tables)
+            Iterable<TpchTable<?>> tables,
+            boolean snapshotIsolationEnabled)
```

Comment:
Since `snapshotIsolationEnabled` is on the SQL Server side, it should come after `TestingSqlServer testingSqlServer` parameter, before Trino configuration.